cyctl/cmd/apply.go

```diff
@@ -89,6 +89,44 @@ func applyYaml(yamlFile []byte, config *rest.Config) error {
 		if err != nil {
 			return err
 		}
+
+		if telemetry {
+			if obj.GetKind() == "Deployment" {
+				labels, _, _ := unstructured.NestedMap(obj.Object, "metadata", "labels")
+				if labels != nil {
+					appLabel, _, _ := unstructured.NestedString(labels, "app")
+					if appLabel == "cyclops-ctrl" {
```

Comment:
Can we check for cyclops controller deployment by checking the name? Also, you can check for all of those in the same if statement
```suggestion
		if telemetry && obj.GetKind() == "Deployment" && obj.GetName() == "cyclops-ctrl" {
```